README.md

```diff
@@ -1,65 +1,71 @@
 # Using NixOPS via vpsFree.cz (a NixOS friendly hoster)
 
-The nixops manual can be found [here](https://nixos.org/nixops/manual/) and
-deployment example [here](https://github.com/vpsfreecz/example-nixops-deployment/).
+This guide tries to assemble all relevant information to enable to you to deploy your setup to vpsFree using nixops 2.0 and nix 2.7.
 
-You can also use nixops to configure containers running on vpsFree. nixops extends the declarative configuration capabilities of NixOS to deploy a cluster of NixOS machines.
+If your are new to NixOps and/or vpsFree we recommend reading
+* nixops 2.0 [manual](https://nixops.readthedocs.io/en/latest/)
+* this  deployment example [uptodate version](https://github.com/vpsfreecz/example-nixops-deployment/)
 
-nixops can be installed between system packages using:
+We assume that you have created a VPS from https://vpsadmin.vpsfree.cz using
+its [admin interface](https://vpsadmin.vpsfree.cz)
 
-```nix
-  environment.systemPackages = with pkgs; [
-    nixops
-  ];
-```
-or to a user profile using `nix-env -i nixops` `nixops --version` returns `NixOps 1.7`.
+* Start it and wait till it is finished
+* Check that you can ping it via its IP address. In this example we will use for this IP `1.2.3.4`.
+* Verify that you can enter ssh into this machine without a password using `ssh root@1.2.3.4`
+* Ensure that you have a recent version of nix (we use 2.7, but from 2.5 upwards will probably be okay)
+* Ensure that your `nixops --version` responds with something like `NixOps 2.0.0-pre-7220cbd`
+* Adapt in machines/hello.nix the value of `1.2.3.4` in the targetHost definition
+* Change in machines/hello.nix "your ssh key" to your public key
```

Comment:
The command to clone the repository is below these instructions, so I think either git clone should be above these instructions or move the instructions to change machines/hello.nix below git clone?

Reply:
Thanks for fixing it so quickly!

Added a line in the readme to fetch the actual vpsadminos.nix by hand via wget. No idea/energy/time to find a way on howto integrate this step in the flake.nix.

Tests with my playground went okay. Maybe you will even want to add a remark, that users can use a playground for their first steps. I thinks it is very good idea of vpsfree.cz and it convinced me to try it out.